ResearchKit/Common/ORKStepHeaderView.m

```diff
@@ -114,10 +114,15 @@ - (instancetype)initWithFrame:(CGRect)frame {
             [self addSubview:_instructionLabel];
         }
         
+        [_captionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+        [_instructionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
         [_learnMoreButton setContentHuggingPriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisVertical];
         [_learnMoreButton setContentCompressionResistancePriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisVertical];
         
 #ifdef LAYOUT_DEBUG
+        _captionLabel.backgroundColor = [UIColor yellowColor];
+        _learnMoreButton.backgroundColor = [UIColor blueColor];
+        _instructionLabel.backgroundColor = [UIColor greenColor];
         self.backgroundColor = [[UIColor purpleColor] colorWithAlphaComponent:0.2];
```

Comment:
Can we set these debug background color with alpha 0.2 or 0.3 to be consistent with other LAYOUT_DEBUG code?